app/ai/voice/agents/breeze_buddy/chat/custom_components.py

```
@@ -0,0 +1,85 @@
+    """The CHAMELEON overlay for one session: name → resolved def."""
+    ui_cat = template.configurations.ui_catalog if template.configurations else None
+    names = list(getattr(ui_cat, "custom_components", None) or [])
+    if not names:
+        return {}
+    try:
+        rows = await get_ui_components_by_names(
+            reseller_id=template.reseller_id,
+            merchant_id=template.merchant_id,
+            names=names,
+        )
+    except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)